src/testRunner/parallel/host.ts

```diff
@@ -147,7 +147,7 @@ namespace Harness.Parallel.Host {
                 }
 
                 cursor.hide();
-                readline.moveCursor(process.stdout, -process.stdout.columns!, -this._lineCount);
+                readline.moveCursor(process.stdout, -process.stdout.columns!, -this._lineCount); // eslint-disable-line @typescript-eslint/no-unnecessary-type-assertion

```

Comment:
Un-suppress this lint - I think it was defs node updating under us, and we've since merged https://github.com/microsoft/TypeScript/pull/33784 which just fixed the lint.